crates/oxc_linter/src/rules/eslint/no_throw_literal.rs

```diff
@@ -0,0 +1,197 @@
+use oxc_ast::{
+    ast::{AssignmentOperator, AssignmentTarget, Expression, LogicalOperator},
+    AstKind,
+};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn no_throw_literal_diagnostic(span: Span, is_undef: bool) -> OxcDiagnostic {
+    let message =
+        if is_undef { "Do not throw undefined." } else { "Expected an error object to be thrown." };
+
+    OxcDiagnostic::warn("Disallow throwing literals as exceptions")
```

Comment:
Your help message should be the warning message. Please refer to the diagnostics message part of the "adding a lint rule" guide

Reply:
https://oxc.rs/docs/contribute/linter/adding-rules.html#diagnostics